simulators/lo/w_LO.py

```diff
@@ -93,23 +93,23 @@ def set_w_LO_freq_PolV(self, params):
         return self.ack + self.tail
 
     def get_w_LO_PolH(self):
-        return f'{self.w_lo_freq_polH}' + self.tail
+        return f'{self.w_lo_freq_polH}MHz' + self.tail
 
     def get_w_LO_PolV(self):
-        return f'{self.w_lo_freq_polV}' + self.tail
+        return f'{self.w_lo_freq_polV}MHz' + self.tail
 
     def get_w_LO_Pols(self):
-        return (f'{self.w_lo_freq_polH},'
-        f'{self.w_lo_freq_polV}' + self.tail)
+        return (f'{self.w_lo_freq_polH}MHz,'
+        f'{self.w_lo_freq_polV}MHz' + self.tail)
 
     def get_w_LO_Synths_Temp(self):
-        return (f'C1={self.w_LO_Synths_Temp[0]},'
-        f'C2={self.w_LO_Synths_Temp[1]}' + self.tail)
+        return (f'{self.w_LO_Synths_Temp[0]}C,'
+        f'{self.w_LO_Synths_Temp[1]}C' + self.tail)
 
     def get_w_LO_HKP_Temp(self):
-        return (f'C1={self.w_LO_HKP_Temp[0]},'
-        f'C2={self.w_LO_HKP_Temp[1]}' f'C3={self.w_LO_HKP_Temp[2]}'
-        f'C4={self.w_LO_HKP_Temp[3]}' + self.tail)
+        return (f'{self.w_LO_HKP_Temp[0]}C,'
```

Comment:
If you want to be consistent with the hardware, if I recall correctly, a '+' character is always present in front of any temperature (this was done with the assumption that at room temperature every device is always going to have a temperature higher than 0°C). This only matters if you are considering the '+' character when testing, otherwise I guess you can leave it as it is, my guess is DISCOS won't check this either so it's a non-problem.

Reply:
Since the protocol is a little bit weird, maybe it's best to be as compliant as possible. I will make the change. Just a question: in the case of a temperature lower than 0° (hardware context) the resulting string will be something like +-1°?

Reply:
Yes, but if I remember correctly this is only the case for the housekeeping temperatures. There is a hard-coded '+' in the strings. Even considering this, if the hardware is powered on and answering, it will likely never have a temperature lower than 0°, probably not even lower than 10°. So, although the scenario you described can possibly happen, it's a very remote case.

Reply:
I see your point, thanks.